apps/cnp/sbox/base/descheduler2.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: descheduler2
+  namespace: cnp
+spec:
+  timeout: 20m
+  releaseName: descheduler
+  chart:
+    spec:
+      chart: descheduler
+      version: 0.18.0
```

Comment:
Is there a reason for 0.18? Latest is 0.29.0 https://github.com/kubernetes-sigs/descheduler/blob/master/charts/descheduler/Chart.yaml#L3 -- don't want to test config as an example that might have been broken/changed in more recent releases